### DISCUSSION TITLE
There is no Docker container? Or a guide on how to install it?

### DISCUSSION THREAD

**Discussion by morphinestyle** [dubinc/dub#454] [Q&A]
I would like to install Dub but it seems to be quite complex.
There is no guide, tutorial or video on how to install Dub?
There isn't even a Docker container?

**Comment by tomy0000000** [comment]
There's a [self-hosting guide](https://dub.co/docs/self-hosting/guide) available now. Although you do need
- Vercel
- Tinybird
- Upstash
- Planetscale
- Resend
- (Maybe more platform)

accounts to make this work. In addition, there's lots of tiny problem you might encounter during the setup.

There's been a [fork](https://github.com/Snazzah/stub) trying to solve these problem, but has no progress for a while...
